Moblin/Various/Settings.swift

```diff
@@ -1319,13 +1319,15 @@ class SettingsDebug: Codable {
     var djiDevices: Bool? = false
 }
 
+let netStreamFps = ["25.0", "29.97", "30.0", "50.0", "59.94", "60.0"]
+
 class SettingsRtmpServerStream: Codable, Identifiable {
     var id: UUID = .init()
     var name: String = "My stream"
     var streamKey: String = ""
     var latency: Int32? = defaultRtmpLatency
     var manualFps: Bool? = false
-    var fps: Double? = 30
+    var fps: Double? = 30.0
```

Comment:
Let's keep fps as is, and add a new variable `selectedFps`, which is an enum. Just to make sure no rounding errors occurs.

Reply:
We should also migrate and old setting to the new settings, so users that have configured an FPS will not lose it.

Reply:
done